packages/block-library/src/navigation/editor.scss

```diff
@@ -457,12 +457,6 @@ $color-control-label-height: 20px;
 	}
 }
 
-.has-fixed-toolbar .wp-block-navigation__responsive-container.is-menu-open {
```

Comment:
It's been a while, but I believe that these styles are probably related to when the overlay menu is open. It may be fine to remove if this still tests well, regardless there should be an incoming overlay-to-template-part PR that would most likely make it redundant.

Reply:
Yes, I tried this flow, and didn't notice any difference with or without the style.